films/views.py

```diff
@@ -1,3 +1,25 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+from django.db.models import Avg
+
+from films.models import Film
+from users.models import Rate
+from core.utils   import token_decorator
+
+class FilmMainView(View):
```

Comment:
마찬가지로 Class 이름에서도 Main 단어를 지워주세요~
```suggestion
class FilmView(View):
```